pkg/controller/installation/core_controller.go

```diff
@@ -1356,6 +1350,8 @@ func (r *ReconcileInstallation) Reconcile(ctx context.Context, request reconcile
 		instance.Status.ImageSet = imageSet.Name
 	}
 	instance.Status.Computed = &instance.Spec
+	instance.Status.Conditions = r.status.Conditions()
```

Comment:
Maybe I am wrong, but I think you want to get from r.status.Conditions, the condition for "Progressing".

If progressing == true, then instance.status.conditions[Progressing] should be set to true and  instance.status.conditions[Available] should be set to false. And vice versa.